src/methods/modular_programs/easy/q7/TwoMethods.java

```diff
@@ -0,0 +1,40 @@
+package methods.modular_programs.easy.q7;
+
+/*
+Write a Java Program that asks the user for two double variables and performs two instructions with them,
```

Comment:
Change question text to:
`
Write a Java Program that asks the user for two double variables and performs two instructions with them, using two methods. Firstly, take the first variable and raise it to the second variable. Secondly, find the larger double variable and convert that double to radians (assuming it is in degrees). 
`